### PR TITLE
Invert nux info on new controls, so new users are not bothered

### PR DESCRIPTION
**Pull request by fm3**
Only existing users should see the NUX dialog presenting the new context menu. New users should just see the new controls without having to choose first.

### URL of deployed dev instance (used for testing):
- https://nuxmoderncontrolsinverse.webknossos.xyz

### Steps to test:
- sign up on dev instance, you should not see the new controls banner

- check out master, sign up, don’t click on the modal
- run the evolution, check out branch, you should see the modal

- refresh db, check out master, sign up, do click on the modal
- run the evolution, check out branch, you should not see the modal again

------
- [x] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/oxalis/view/novel_user_experiences/01-present-modern-controls.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function PresentModernControls() {
   const dispatch = useDispatch();
   const activeUser = useSelector(state => state.activeUser);
   const [isModalVisible, setIsModalVisible] = React.useState(
-    activeUser != null && !activeUser.novelUserExperienceInfos.hasSeenModernControlsModal,
+    activeUser != null && !activeUser.novelUserExperienceInfos.shouldSeeModernControlsModal,
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/view/novel_user_experiences/01-present-modern-controls.js:13]: I think this is why I currently see the modal even when signing up on the dev instance.

```suggestion
    activeUser != null && activeUser.novelUserExperienceInfos.shouldSeeModernControlsModal,
```

**Review by daniel-wer — COMMENTED** [review]
Doesn't seem to work as intended yet, I gave a suggestion for a fix :)

**Comment by fm3** [comment]
Thanks for your commit, my bad with the missing `!` 🙈 Could you approve the final version so it can be merged?

**Review by daniel-wer — APPROVED** [review]
Code LGTM, I'd do a quick final test once CircleCI is back up.

**Pull request merged**